[BUG] Issue with the pipeline check

I got a report/question by email about a niche bug that occurred in the `processing_pipeline_check` function. They were trying to just run 2d image tracking, and so had unchecked `Run 3d triangulation`. However, that gave them this error, which I reproduced and pasted down below:

```
Number of `.mp4`'s in C:\Users\aaron\FreeMocap_Data\recording_sessions\freemocap_sample_data\synchronized_videos: 3
2024-07-24 10:44:10,843 - freemocap.data_layer.recording_models.recording_info_model - INFO - Number of `.mp4`'s in C:\Users\aaron\FreeMocap_Data\recording_sessions\freemocap_sample_data\synchronized_videos: 3
[2024-07-24T10:44:10.844817] [Δt:0.000801s] [   DEBUG] [freemocap.data_layer.recording_models.recording_info_model] [recording_info_model:get_number_of_frames_in_videos():286] [PID:72676:Process-1 TID:76512:MainThread ] No 'timestamps' 
directory, finding frame counts from synchronized videos
[2024-07-24T10:44:10.866817] [Δt:0.025000s] [   ERROR] [freemocap.core_processes.process_motion_capture_videos.process_recording_folder] [process_recording_folder:process_recording_folder():64] [PID:72676:Process-1 TID:76512:MainThread 
] processing parameters are not valid for recording status
2024-07-24 10:44:10,866 - freemocap.core_processes.process_motion_capture_videos.process_recording_folder - ERROR - processing parameters are not valid for recording status
Process Process-1:
Traceback (most recent call last):
  File "c:\Users\aaron\anaconda3\envs\freemocap_dev_2\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
message: [2024-07-24T10:44:10.0866] [ERROR] [freemocap.core_processes.process_motion_capture_videos.process_recording_folder] [Process:72676 Thread:76512] ::::: [freemocap.core_processes.process_motion_capture_videos.process_recording_folder:process_recording_folder():64] |||| processing parameters are not valid for recording status.
  File "c:\Users\aaron\anaconda3\envs\freemocap_dev_2\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\aaron\Documents\GitHub\freemocap\freemocap\core_processes\process_motion_capture_videos\process_recording_folder.py", line 67, in process_recording_folder
    raise e
  File "C:\Users\aaron\Documents\GitHub\freemocap\freemocap\core_processes\process_motion_capture_videos\process_recording_folder.py", line 62, in process_recording_folder
    processing_pipeline_check(processing_parameters=recording_processing_parameter_model)
  File "C:\Users\aaron\Documents\GitHub\freemocap\freemocap\core_processes\process_motion_capture_videos\processing_pipeline_functions\pipeline_check.py", line 24, in processing_pipeline_check
    raise FileNotFoundError(
FileNotFoundError: No 3d data found at: C:\Users\aaron\FreeMocap_Data\recording_sessions\freemocap_sample_data\output_data\mediapipe_skeleton_3d.npy
```

The issue seems to be that if you're running a completely new recording and only enable 2D image tracking, this section will run:
```
if not processing_parameters.anipose_triangulate_3d_parameters_model.run_3d_triangulation:
        if not status_check_dict["data3d_status_check"]:
            raise FileNotFoundError(
                f"No 3d data found at: {processing_parameters.recording_info_model.data_3d_npy_file_path}"
            )
 ```
 
 And will raise that `FileNotFoundError`. 
 
 My initial question is, is there any reason we raise an error about the 3D data not existing if `Run 3d triangulation` is disabled? The case of people wanted to just do 2d image tracking is a pretty niche one IMO, but I'm just wondering about this on a more general scale of why we do this specific search for the 3d file at this specific juncture?

--- a/freemocap/core_processes/process_motion_capture_videos/processing_pipeline_functions/pipeline_check.py
+++ b/freemocap/core_processes/process_motion_capture_videos/processing_pipeline_functions/pipeline_check.py
@@ -17,7 +17,7 @@ def processing_pipeline_check(processing_parameters: ProcessingParameterModel) -
             )
 
     if not processing_parameters.anipose_triangulate_3d_parameters_model.run_3d_triangulation:
-        if not status_check_dict["data3d_status_check"]:
+        if not status_check_dict["data3d_status_check"] and not status_check_dict["single_video_check"]:
             raise FileNotFoundError(
                 f"No mediapipe 3d data found at: {processing_parameters.recording_info_model.mediapipe_3d_data_npy_file_path}"
             )

